tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot

```diff
@@ -74,6 +74,27 @@ Create VCH - target URL
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - thumbprint verification
+    Log To Console  \nRunning vic-machine create - thumbprint verification
+    Set Test Environment Variables
+    # TODO: need an integration/vic-test update to include the about.cert command
+    #${rc}  ${thumbprint}=  Run And Return Rc And Output  govc about.cert -k | jq -r .ThumbprintSHA1
+    ${rc}  ${thumbprint}=  Run And Return Rc And Output  openssl s_client -connect $(govc env -x GOVC_URL_HOST):443 </dev/null 2>/dev/null | openssl x509 -fingerprint -noout | cut -d= -f2
```

Comment:
You should be able to use `%{TEST_URL}` instead of the inlined `govc env` comment